perf/perf_rawevents.py

```diff
@@ -72,7 +75,11 @@ def setUp(self):
         # Clear the dmesg to capture the delta at the end of the test.
         process.run("dmesg -C")
 
-    def run_event(self, filename, perf_flags):
+    def run_event(self, filename, eventname):
+        if eventname == 'raw':
+            perf_flags = "perf stat -e r"
+        elif eventname == 'name':
+            perf_flags = "perf stat -e "
```

Comment:
This code can be simplified by removing one check.

perf_flags = "perf stat -e"
if eventname == 'raw':
     perf_flags = "perf stat -e r"